llvm/lib/Target/AArch64/GISel/AArch64InstructionSelector.cpp

```diff
@@ -597,8 +597,14 @@ getRegClassForTypeOnBank(LLT Ty, const RegisterBank &RB,
 /// Given a register bank, and size in bits, return the smallest register class
 /// that can represent that combination.
 static const TargetRegisterClass *
-getMinClassForRegBank(const RegisterBank &RB, unsigned SizeInBits,
+getMinClassForRegBank(const RegisterBank &RB, TypeSize SizeInBits,
```

Comment:
The variable name "SizeInBits" is not appropriate for Type "TypeSize". Please have something else.

Reply:
I think it's fine, as it's obtained from `TypeSize RegisterBankInfo::getSizeInBits()`. 

There are both `TypeSize LLT::getSizeInBytes()` and `TypeSize LLT::getSizeInBits()` so `TypeSize` itself does not have a unit.